src/napari_omero/widgets/thumb_grid.py

```diff
@@ -2,12 +2,79 @@
 
 from qtpy.QtCore import QSize, Qt
 from qtpy.QtGui import QIcon, QImage, QPixmap
-from qtpy.QtWidgets import QListWidget, QListWidgetItem
+from qtpy.QtWidgets import (
+    QLabel,
+    QListWidget,
+    QListWidgetItem,
+    QSizePolicy,
+    QTableWidget,
+    QTableWidgetItem,
+    QVBoxLayout,
+    QWidget,
+)
 
 from .gateway import QGateWay
 from .tree_model import OMEROTreeItem
 
-THUMBSIZE = 96
+THUMBSIZE = 192
+
+
+class ThumbItemWidget(QWidget):
+    """Custom widget that displays an icon with a table of properties below."""
+
+    def __init__(self, icon: QIcon, wrapper, parent=None):
+        super().__init__(parent)
+        self.wrapper = wrapper
+
+        layout = QVBoxLayout(self)
+
+        # Icon label
+        icon_label = QLabel()
+        icon_label.setPixmap(icon.pixmap(THUMBSIZE, THUMBSIZE))
```

Comment:
```suggestion
        pixmap = icon.pixmap(THUMBSIZE, THUMBSIZE)
        icon_height = pixmap.height()
        icon_label.setPixmap(pixmap)
```
Let's get the height of the actual icon, so we can use that for the height below.